Buildings/Applications/DHC/EnergyTransferStations/CoolingDirectControlledReturn.mo

```diff
@@ -0,0 +1,396 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingDirectControlledReturn
+  "Direct cooling ETS model for district energy systems with in-building 
+    pumping and controlled district return temperature"
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    redeclare final package Medium1 = Medium,
+    redeclare final package Medium2 = Medium,
+    final m1_flow_nominal = mDis_flow_nominal,
+    final m2_flow_nominal = mBui_flow_nominal,
+    show_T = true);
+
+ replaceable package Medium =
+   Modelica.Media.Interfaces.PartialMedium "Medium in the component";
+
+  // mass flow rates
+  parameter Modelica.SIunits.MassFlowRate mDis_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of district cooling side";
+
+  parameter Modelica.SIunits.MassFlowRate mBui_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of building cooling side";
+
+  parameter Modelica.SIunits.MassFlowRate mByp_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate through the bypass segment";
+
+  // pressure drops
+  parameter Modelica.SIunits.PressureDifference dpConVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop in the control valve";
+
+  parameter Modelica.SIunits.PressureDifference dpCheVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop in the check valve";
+
+  // Controller parameters
```

Comment:
I agree reverse action and output signal range (yMin, yMax) are inherent and will remove these parameters. However I think the controllerType and PID tuning parameters (time constants, weights, etc.) should remain, since users may need to adjust these depending on their fluid network. If there are any other control parameters you think users do not need to modify, please let me know which ones and/or we can discuss.